parser.js

```diff
@@ -21,7 +21,7 @@ const BOM = createToken({
 const NewLine = createToken({
   name: 'NewLine',
   // eslint-disable-next-line no-control-regex
-  pattern: /\x0D\x0A|[\x0A\x0C\x85\u2028\u2029]/
+  pattern: /\x0D\x0A|[\x0A\x0D\x0C\x85\u2028\u2029]/
```

Comment:
Also, sorry to be pedantic but could you insert it after `\x0C` to keep it ordered?

Reply:
Yep, thought I had ordered it